api/api_ethereum.go

```diff
@@ -305,7 +307,7 @@ func (api *EthereumAPI) MaxPriorityFeePerGas(ctx context.Context) (*hexutil.Big,
 	return api.publicKlayAPI.GasPrice(ctx)
 }
 
-type feeHistoryResult struct {
+type FeeHistoryResult struct {
```

Comment:
Why did you change this?

Reply:
which test?

Reply:
RPC test..! 
You can test it by building ken and just call feeHistoryAPI via http.